src/ape_accounts/_cli.py

```diff
@@ -157,6 +157,17 @@ def _import(cli_ctx, alias, import_from_mnemonic, custom_hd_path):
     )
 
 
+@cli.command(name="export", short_help="Unlock an account and export the private key")
```

Comment:
Don't need to do this because `export` is not a python reserved keyword
```suggestion
@cli.command(short_help="Unlock an account and export the private key")
```